macro/Version.py

```diff
@@ -47,11 +47,10 @@
 def execute(macro, args):
     if args:
         version = str(args)
-        if version.lower() == 'm-turtle':
-            # TODO Change when m-turtle name is decided
+        if version.lower() == 'melodic':
             return ('<span style="background-color:#FFFF00; '
                     'font-weight:bold; padding: 3px;">'
-                    'Expected in M-Turtle</span>')
+                    'Expected in Melodic Morenia</span>')
```

Comment:
This page uses `...  <<Version(ROS Kinetic)>>`:

http://wiki.ros.org/rosbag/Commandline#record

Which renders as `New in ROS Kinetic`, so... This logic seems fragile or naive. Maybe it should be something like "if m-turtle or melodic in name, do 'Expected in {version}', else do 'New in {version}'".